src/system/chaos.291

```diff
@@ -2426,29 +2426,49 @@ BBLK
 	 JRST [	AOS CHNLOS	;No buffers, lose this packet
 		JRST CHXBK1 ]
 	HRLI A,-%CPKDT		;Copy out the packet header
+IFN KLH10P,[
```

Comment:
This needs a closing `]`.  Or the line could just be removed because the comment doesn't need to be inside a conditional.